src/devices/video/topcat.h

```diff
@@ -14,9 +14,16 @@
 #define MCFG_TOPCAT_PLANEMASK(_mask) \
 	downcast<topcat_device &>(*device).set_planemask(_mask);
 
+#define MCFG_TOPCAT_INT_CB(_write)                               \
```

Comment:
We're deperecating this style of callback configuration - please use devcb3 style bind targets.  See e.g. recent changes by @Osso13 for examples of migration.